modules/internal/Bytes.chpl

```diff
@@ -895,8 +895,9 @@ module Bytes {
                   - `decodePolicy.escape` escapes each illegal byte with
                     private use codepoints
 
-    :throws DecodeError: if `decodePolicy.strict` is passed to the `policy`
-            argument and the :type:`bytes` contains non-UTF-8 characters.
+    :throws: Throws a :class:`~Errors.DecodeError` if `decodePolicy.strict` is
+      passed to the `policy` argument and the :type:`bytes` contains non-UTF-8
+      characters.
```

Comment:
In the built docs, it now shows as "throws Throws ...". Can you remove the extra "Throws" here and make it `:throws: a :class:...`? Ditto for all the other cases in `String.chpl`

Reply:
Yeah I can change that. I was using the IO module's style of `:throws: Throws a ...`, but it looks like the string and bytes docs don't do that so I'll change it.